pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -610,7 +610,10 @@ func parseSystemAdapterConfig(config *zconfig.EdgeDevConfig,
 	portConfig.Ports = newPorts
 
 	// Any content change?
-	if cmp.Equal(getconfigCtx.devicePortConfig.Ports, portConfig.Ports) &&
+	// Should we exclude changes in ParseError and ParseErrorTime?
+	// XXX Equal functions skips those
+	if getconfigCtx.devicePortConfig.Equal(portConfig) &&
+		// XXX if cmp.Equal(getconfigCtx.devicePortConfig.Ports, portConfig.Ports) &&
```

Comment:
Ah.. This is the Crux of the issue - Is it?
Then we should remove them from the struct??

Reply:
But we want to be able to pass the ParseErrors back to zedcloud, and today we send back the DevicePortConfigList.

I think I want to rework this to:
 - have zedagent publish the above even if just the ParseError/ParseErrorTime changed (i.e., revert this back to DeepEqual)
 - instead of having nim ignore DPCs with no management address, have it add them to the DevicePortConfigList, but then have it skip them immediately when determining what to test.

That ensures that we will still report the ParseErrors, and if someone sets the UI to have no management ports, the currentIndex as reported by the device will be non-zero indicating that the highest priority entry is skipped.

Reply:
I've updated based on that approach.